src/cdfDatasource.ts

```diff
@@ -156,12 +160,14 @@ export async function getTimeseries(
         data,
         method,
         path: `/timeseries/byids`,
+        cacheTime: '1h',
       });
     } else {
       items = await connector.fetchAndPaginate({
         data,
         method,
         path: `/timeseries/list`,
+        cacheTime: '1h',
```

Comment:
can we have lower value here, cause during `1h` some ts might be added that will match provided filters

Reply:
fixed, now it's 151 seconds (prime number so they don't overlap)